applications/API/forms.py

```
@@ -0,0 +1,406 @@
+    """
+    
+    # Dynamic choices will be set in __init__
+    insult = forms.ChoiceField(
+        widget=ModelSelect2Widget(
+            model=MyOtherModel,
+            search_forms=['insult_reference_id__icontains'],
+            set_to_cache = True,
+             max_results = 200                                                                                                          
+        )
+    )
+        required=True, 
```

Pylint (3.3.7)

[error] Parsing failed: 'unexpected indent (git.applications.API.forms, line 269)'

(E0001)